client/rest-high-level/src/test/java/org/elasticsearch/client/IndicesRequestConvertersTests.java

```diff
@@ -868,6 +868,12 @@ private void resizeTest(ResizeType resizeType, CheckedFunction<ResizeRequest, Re
         ResizeRequest resizeRequest = new ResizeRequest(indices[0], indices[1]);
         resizeRequest.setResizeType(resizeType);
         Map<String, String> expectedParams = new HashMap<>();
+
+        if (ESTestCase.randomBoolean()) {
```

Comment:
I see this is in the existing style in this class, but it is at odds with the rest of the codebase where we would simply write `randomBoolean()`.

Reply:
Addressed in https://github.com/elastic/elasticsearch/pull/39752/commits/ce68119b9315a4dc812146e9e40b29cb8f9344f1; in a separate PR we can convert everything in this class to `randomBoolean()` for consistency.